.github/workflows/linux.yml

```diff
@@ -304,7 +304,8 @@ jobs:
           sudo env MAKEFLAGS=-j$(nproc) gem install \
             grntest \
             pkg-config \
-            red-arrow
+            red-arrow \
+            -- --with-cxxflags=-std=c++17
```

Comment:
This should be fixed when red-arrow 10.0.0 is published.
So we don't need to do anything here.